app/agents_v2/primary_agent/troubleshooting/diagnostic_sequencer.py

```
@@ -0,0 +1,512 @@
+        # Find next available step
+        available_steps = [
+            step for step in workflow.diagnostic_steps
+            if step.step_id not in excluded_ids
+        ]
+        
+        if not available_steps:
+            logger.info("No more diagnostic steps available")
+            return None
+        
+        # Filter by current phase and difficulty
+        current_phase = session.current_phase
```

Ruff (0.11.9)

Local variable `current_phase` is assigned to but never used

Remove assignment to unused variable `current_phase`

(F841)